editor/plugins/editor_plugin.cpp

```diff
@@ -507,6 +507,30 @@ bool EditorPlugin::build() {
 	return success;
 }
 
+void EditorPlugin::get_meshlib_item_metadata_list(MeshInstance3D *p_object, List<MeshLibrary::PluginGeneratedMetadata> *p_list) {
+	TypedArray<Dictionary> virtual_list;
+	if (GDVIRTUAL_CALL(_get_meshlib_item_metadata_list, p_object, virtual_list)) {
+		for (int i = 0; i < virtual_list.size(); i++) {
+			Dictionary virtual_entry = virtual_list.get(i);
```

Comment:
```suggestion
		for (const Dictionary &virtual_entry : virtual_list) {
```
Don't do random access operations on `List` when avoidable